### DISCUSSION TITLE
Looking for full stack dev

### DISCUSSION THREAD

**Discussion by technerdfromhellv1** [community/community#52755] [Other Feature Feedback, Questions, & Ideas] [answered]

**Comment by adlai** [comment]
Yeah, it's probably misplaced; good luck, anyway!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @technerdfromhellv1 , we're happy you're here 😀

However, we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
